### ISSUE TITLE
Training loss function

### ISSUE THREAD

**Issue by prach6i** [jzhoubu/vsearch#4]
Can you please share the code for the loss used in the training. Also, in the code you have not used the cts mask, can you explain how is it used during the training?

**Comment by jzhoubu** [comment]
Hi, @prach6i. The CTS mask is employed exclusively during the training, and it is not utilized during inference. You can find the pseudocode for the CTS mask in our paper, which should help you implement it. Please feel free to share any specific issues or questions you encounter. We're here to help.

**Comment by jzhoubu** [comment]
Hi @prach6i 

We have recently released the training pipeline for our repo. The code for the loss function can be found in the `src/vdr/training/loss.py` [here](https://github.com/jzhoubu/VDR/blob/master/src/vdr/training/loss.py).
